SupportFiles/UrlChecker.py

```diff
@@ -84,66 +153,71 @@ def main():
-							badUrl = False
-							url, code, message = '', '', ''							
-							try:
+		# For each file, searches the file for references to http and https.
+		for currentFile in matchingFileList:
+			try:
+				fileCount += 1
+				print('Checking File #' + str(fileCount) + ', file: ' + currentFile)
+				searchfile = open(currentFile, 'r')
+				lineNumber = 0
+				newUrlFile = True
+				commentBlock = False
+				for line in searchfile:
+					lineNumber += 1
+
+					###################################
+					# Ignore comments/comment blocks
+					if '<!--' in line:
+						# debug: print('Start of Comment Block at Line #:'  + str(lineNumber) + ': ' + line)
+						commentBlock = True
+
+					if commentBlock and ('-->' in line):
+						# debug: print('End of Comment Block at Line #:'  + str(lineNumber) + ': ' + line)
+						line = line.split('-->')[1]  # TRICKY: still check this line, just in case stuff after comment
+						commentBlock = False
+
+					if commentBlock:
+						# debug: print('Skipping Comment Line #:'  + str(lineNumber) + ': ' + line)
+						continue
+					###################################
+
+					if 'href="/' in line:
+						if hRefForLocalLinks is not None:
+							oldLine = line
+							line = line.replace('href="/', hRefForLocalLinks)
+							# debug: print('Replaced local link line-Original: ' + oldLine + 'Replaced: ' + line)
+
+					if 'http' in line:
+						# Get a list of items between quotes " "
+						quotedItems = re.findall('"([^"]*)"', line)
+						for quotedItem in quotedItems:
+							# If Web Link found, then test URL
+							if ('http' or 'https') in quotedItem:
 								url = quotedItem
-								response = urllib2.urlopen(url)
-								code = str(response.getcode())
-								checkUrl = response.geturl() # just in case we want to check the 2
-								message = 'no error'
-
-							except urllib2.HTTPError as httpErr:
-								badUrl  = True
-								code    = str(httpErr.code)
-								# message = TODO
-								# if more info needed then potentially strip from response
-								# error_message = httpErr.read()
-								# print(error_message)
-					
-							except urllib2.URLError as urlErr:
-								badUrl  = True
-								code    = '0'
-								message = urlErr.args
-
-							if badUrl or verbose :
-								urlCount += 1
-								if newUrlFile : 
-									urlFileCount += 1
-									newUrlFile = False
-								urlError = [str(urlCount), str(urlFileCount), currentFile, str(lineNumber), url, code, message]
-								urlErrorsList.append(urlError)
-
-			searchfile.close
-
-		if verbose: 
-			print(str(urlCount) + ' URLs found in ' + str(fileCount) + ' files checked.')			
-		else : 						
-			if urlCount == 0 :			
+								badUrl, code, message = checkUrl(url)
+
+								if badUrl or verbose:
+									urlCount += 1
+									if newUrlFile:
+										urlFileCount += 1
+										newUrlFile = False
+									urlError = [str(urlCount), str(urlFileCount), currentFile, currentFile, str(lineNumber), url, code, message]
```

Comment:
@ACueva - I just noticed this in the output csv, but in your pylinting splendor, you repeated "currentFile" twice here in error, this should be fixed before PR is merged